canonical/ubuntu.com
# [WD-7737] desktop features rebrand

## Done

- Rebranded /desktop/features according to [Figma design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?type=design&node-id=888-10097&mode=design&t=g2XHjM3FGL8B1bSo-0)

## QA

- Navigate to /desktop/features and check the page
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7737

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
